src/main/java/roomescape/auth/AuthController.java

```diff
@@ -0,0 +1,66 @@
+package roomescape.auth;
+
+import jakarta.servlet.http.Cookie;
+import jakarta.servlet.http.HttpServletRequest;
+import jakarta.servlet.http.HttpServletResponse;
+import jakarta.validation.Valid;
+import org.springframework.http.ResponseEntity;
+import org.springframework.stereotype.Controller;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import roomescape.global.exception.RoomescapeUnauthorizedException;
+
+
+@Controller
+public class AuthController {
+
+    private final AuthService authService;
+
+    public AuthController(AuthService authService) {
+        this.authService = authService;
+    }
+
+    @PostMapping("/login")
+    public ResponseEntity login(@RequestBody @Valid LoginRequest loginRequest, HttpServletResponse response) {
```

Comment:
```suggestion
    public ResponseEntity<Void> login(@RequestBody @Valid LoginRequest loginRequest, HttpServletResponse response) {
```

물론 미션에서 주어진 코드들이 이렇게 되어있지만 그래도 타입을 지정하는 방식으로 전부 수정해주는 건 어떨까요?😊

Reply:
동의합니다 ! 반영했어요 👍